src/NosCore.GameObject/Character.cs

```diff
@@ -877,44 +879,12 @@ public FdPacket GenerateFd()
             return new FdPacket
             {
                 Reput = Reput,
-                Dignity = (int)Dignity,
-                ReputIcon = GetReputIco(),
-                DignityIcon = Math.Abs(GetDignityIco())
+                Dignity = Dignity,
+                ReputIcon = (int)ReputIcon, //todo change packet type
+                DignityIcon = (int)DignityIcon //todo change packet type
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5574106)